src/otx/algorithms/detection/configs/detection/cspdarknet_yolox_s/deployment.py

```diff
@@ -0,0 +1,13 @@
+"""MMDeploy config of YOLOX_S model for Detection Task."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+_base_ = ["../../base/deployments/base_detection_dynamic.py"]
+
+ir_config = dict(
+    output_names=["boxes", "labels"],
+)
+
+backend_config = dict(
+    model_inputs=[dict(opt_shapes=dict(input=[-1, 3, 416, 416]))],
```

Comment:
Do you know why in YOLOX train in 640x640 but infer in 416x416?
c.c. @eunwoosh

Reply:
backend_config.model_inputs will be configured from data_pipeline.py. Therefore exported model will inference on 640*640 images. However YOLOX-TINY, the existed one, resize 416 * 416 for test data pipeline, so YOLOX-TINY exported model will inference on 416 * 416 image. I guess this setting is for faster inference since YOLOX-TINY is Geti's speed model.
Anyway @goodsong81 do you think this file should be modified even if it will automatically be patched?

Reply:
I think the reason why YOLOX tiny uses 416 input size in test pipeline is because config was just copied from mmdet. I don't know why mmdet uses that input size, but it's weird to use 640 input size when training and use 412 at inference. Anyway, I think although it's overwritten by 'data_pipeline.py', but changing it as 640 can be better to avoid possible confusion later.